File: nodejs8/handler.js
'use strict';

function fib(n) {
  if (n == 1 || n == 2) {
    return 1;
  }
  if (n == 3) {
    return 2;
  }
  return fib(n - 1) + fib(n - 2);
}

module.exports.fibonacci = (event, context, callback) => {
  const response = {
    statusCode: 200,
    body: fib(35).toString()
  };

  callback(null, response);
};

File: python3/handler.py
import json

def fib(n):
    if n == 1 or n == 2:
        return 1
    if n == 3:
        return 2
    return fib(n-1) + fib(n-2)

def fibonacci(event, context):
    return {
        "statusCode": 200,
        "body": str(fib(35))
    }
